src/shipit_uplift/shipit_uplift/coverage.py

```diff
@@ -94,12 +101,16 @@ class CodecovCoverage(Coverage):
     URL = 'https://codecov.io/api/gh/marco-c/gecko-dev'
 
     @staticmethod
-    @lru_cache(maxsize=32)
+    def _get(url):
+        return requests.get(CodecovCoverage.URL + url + '?access_token=%s' % secrets.CODECOV_TOKEN)
```

Comment:
I would rather use a `.format` here to increase readability.

Reply:
Done in the latest commit.